plugins/torrent-tracker.js

```diff
@@ -0,0 +1,34 @@
+var Server = require('bittorrent-tracker').Server 
+
+function isFunction (f) {
+  return 'function' === typeof f
+}
+
+function isString (s) {
+  return 'string' === typeof s
+}
+
+exports.name = 'torrent-tracker'
+exports.version = '1.0.0'
+
+//trust-proxy 'x-forwarded-for' header from reverse proxy 
+//tell clients to announce on this interval (ms) 
+exports.manifest = {
```

Comment:
Oh, you know, this isnt how the manifest should be used. This looks like you're using it for config. The manifest is actually a listing of the methods in the plugin which are RPC-accessable, in the form:
```
{
   methodName: type
}
```
Where `type` can be `'sync'`, `'async'`, `'source'`, `'sink'`, and `'duplex'`. We need to document those, but here it is briefly:

- async: last param is a `cb(err, ...)`
- sync: constructed like a typical sync function, which is handy if called locally by sbot, but the rpc will convert it into an async behavior. If it throws, will `cb(exception)`. If it returns, will `cb(null, returnvalue)`
 - the other three: pull-stream types

You dont have any methods yet, so the manifest should just be empty.

@dominictarr where should plugin config go? For now, I think just stick it on some other object.